sys/dev/mrsas/mrsas_ioctl.h

```diff
@@ -88,7 +88,6 @@ __FBSDID("$FreeBSD$");
 
 #define	MEGAMFI_RAW_FRAME_SIZE	128
 
-#pragma pack(1)
```

Comment:
Perhaps you could #ifdef this as something like:

```
#if <use non-CHERI ABI>
#pragma pack(1)
#endif
```

Presumably leaving the bare `#pragma pack()` without the #if wouldn't do any harm.  This would leave a smaller diff relative to upstream.